src/examples/TemplateExamplePage.tsx

```diff
@@ -0,0 +1,47 @@
+'use strict';
+import {StyleSheet, View, Text, Platform} from 'react-native';
+import React, {useState} from 'react';
+import Clipboard from '@react-native-community/clipboard';
+import styles from './ExamplePageStyleSheet';
+// Add import for page's component here
+
+// Replace TemplateExamplePage with <ComponentName>ExamplePage
+export const TemplateExamplePage: React.FunctionComponent<{}> = () => {
+    // Replace with string version of JSX snippet used to render component for example1
+    const example1jsx = "";
+    // Replace with string version of JSX snippet used to render component for example2
+    const example2jsx = "";
+
+    const copyExample1ToClipboard = () => {
+      Clipboard.setString(example1jsx);
+    };
+
+    const copyExample2ToClipboard = () => {
+      Clipboard.setString(example2jsx);
+    };
+    
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>Template</Text>
+      <Text style={styles.description}>Add component description here. See XAML Controls Gallery for description contents if available.</Text>
```

Comment:
>XAML Controls Gallery [](start = 75, length = 21)

Can this be a hyperlink using the Linking module?  RNW does appear to support it.  See Linking module docs:  [https://reactnative.dev/docs/linking](https://reactnative.dev/docs/linking)

Reply:
Yea I'll have to investigate it we can get hyperlinks at this time. I recall Jon saying something before about that not working. I'll look into it next.